samples/snippets/sample_default_values.py

```diff
@@ -0,0 +1,119 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample script showing how to handle default values when communicating
+with the Compute Engine API.
+"""
+# [START compute_instances_verify_default_value]
+# [START compute_usage_report_set]
+# [START compute_usage_report_get]
+# [START compute_usage_report_disable]
+from google.cloud import compute_v1
+# [END compute_usage_report_disable]
+# [END compute_usage_report_get]
+# [END compute_usage_report_set]
+
+
+# [START compute_usage_report_set]
+def set_usage_export_bucket(project_id: str, bucket_name: str,
+                            report_name_prefix: str = "") -> None:
+    """
+    Set Compute Engine usage export bucket for the Cloud project.
+    This sample presents how to interpret the default value for the
+    report name prefix parameter.
+
+    Params:
+        project_id: project ID or project number of the project to update.
+        bucket_name: Google Cloud Storage bucket used to store Compute Engine
+            usage reports. An existing Google Cloud Storage bucket is required.
+        report_name_prefix: Prefix of the usage report name which defaults to an empty string
+            to showcase default values behaviour.
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
+
+    if not report_name_prefix:
+        # Sending an empty value for report_name_prefix results in the
+        # next usage report being generated with the default prefix value
+        # "usage_gce". (ref: https://cloud.google.com/compute/docs/reference/rest/v1/projects/setUsageExportBucket)
+        print("Setting report_name_prefix to empty value causes the report "
+              "to have the default prefix of `usage_gce`.")
+
+    projects_client = compute_v1.ProjectsClient()
+    operation = projects_client.set_usage_export_bucket(
+        project=project_id, usage_export_location_resource=usage_export_location)
+
+    op_client = compute_v1.GlobalOperationsClient()
+    op_client.wait(project=project_id, operation=operation.name)
+    return
+# [END compute_usage_report_set]
+
+
+# [START compute_usage_report_get]
+def get_usage_export_bucket(project_id: str) -> compute_v1.UsageExportLocation:
+    """
+    Retrieve Compute Engine usage export bucket for the Cloud project.
+    Replaces the empty value returned by the API with the default value used
+    to generate report file names.
+
+    Params:
```

Comment:
```suggestion
    Args:
```
Please follow https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings for docstring style.

Reply:
OK.